cli/tauri.js/src/types/cargo.ts

```diff
@@ -1,3 +1,13 @@
+export interface CargoToml {
```

Comment:
CargoToml was replaced by `CargoManifest`, seems like this was a wrong merge

Reply:
Ok, will fix.